### ISSUE TITLE
output key in synchronization module

### ISSUE THREAD

**Issue by robertoostenveld** [eegsynth/eegsynth#45]
the handling of the output prefix/name in the synchronization module is not consistent with other modules

**Comment by robertoostenveld** [comment]
```
mac011> grep key.*prefix */*.py
brain/brain.py:                key = "%s.%s.%s" % (config.get('output','prefix'), chan, band)
eyeblink/eyeblink.py:    key = "%s.channel%d" % (config.get('output','prefix'), channel)
heartrate/heartrate.py:    key = "%s.channel%d" % (config.get('output','prefix'), channel)
inputmidi/inputmidi.py:            key = "{}.control{:0>3d}".format(config.get('output', 'prefix'), msg.control)
inputmidi/inputmidi.py:            key = "{}.note{:0>3d}".format(config.get('output','prefix'), msg.note)
inputmidi/inputmidi.py:            key = "{}.note".format(config.get('output','prefix'))
inputosc/inputosc.py:    key = config.get('output', 'prefix')+addr.replace('/', '.')
keyboard/keyboard.py:                    key = "{}.note".format(config.get('output','prefix'))
keyboard/keyboard.py:                    key = "{}.note{:0>3d}".format(config.get('output','prefix'), msg.note)
launchcontrol/launchcontrol.py:            key = "{}.control{:0>3d}".format(config.get('output', 'prefix'), msg.control)
launchcontrol/launchcontrol.py:                key = "{}.note{:0>3d}".format(config.get('output','prefix'), msg.note)
launchcontrol/launchcontrol.py:                key = "{}.note".format(config.get('output','prefix'))
muscle/muscle.py:            key = "%s.%s" % (config.get('output','prefix'), name)
sequencer/sequencer.py:        key = "{}.note".format(config.get('output','prefix'))
synchronization/synchronization.py:key = config.get('output','prefix')
```

**Comment by robertoostenveld** [comment]
mac011> git commit synchronization/
[master 7d1e524] FIX see issue #45
 2 files changed, 6 insertions(+), 10 deletions(-)
